src/tcgwars/logic/impl/gen2/AquapolisNG.groovy

```diff
@@ -0,0 +1,2961 @@
+          text "Once during your turn (before your attack), if Hypno is your Active Pokémon, you may use this power. The Defending Pokémon is now Asleep. This power can't be used if Hypno is affected by a Special Condition."
+          actionA {
+          }
+        }
+        move "Spiral Aura", {
+          text "30 damage. Unless this attack Knocks Out the Defending Pokémon, if your opponent has any Benched Pokémon you may choose 1 of them and switch it with the Defending Pokémon. (Do the damage before switching the Pokémon.)"
+          energyCost P, C, C
+          attackRequirement {}
+          onAttack {
+            damage 30
+          }
+        }
+      };
+      case JUMPLUFF_17:
+      return evolution (this, from:"Skiploom", hp:HP070, type:G, retreatCost:null) {
+        weakness R
+        resistance W, MINUS30
+        pokeBody "Fluff", {
+          text "During your opponent's turn, if Jumpluff would be damaged or affected by an opponent's attack and it already has at least 1 damage counter on it, flip a coin. If heads, prevent all effects of that attack (including damage)."
+          delayedA {
+          }
+        }
+        move "Cotton Punch", {
+          text "30x damage. Flip 2 coins. This attack does 30 damage times the number of heads."
+          energyCost G
+          attackRequirement {}
+          onAttack {
+            damage 30
+          }
+        }
+      };
+      case JYNX_18:
+      return basic (this, hp:HP060, type:P, retreatCost:2) {
+        weakness P
+        move "Meditate", {
+          text "10+ damage. This attack does 10 damage plus 10 more damage for each damage counter on the Defending Pokémon."
+          energyCost P, C
+          attackRequirement {}
+          onAttack {
+            damage 10
+          }
+        }
+        move "Confuse Ray", {
+          text "20 damage. Flip a coin. If heads, the Defending Pokémon is now Confused."
+          energyCost P, C, C
+          attackRequirement {}
+          onAttack {
+            damage 20
+          }
+        }
+      };
+      case KINGDRA_19:
+      return evolution (this, from:"Seadra", hp:HP120, type:W, retreatCost:3) {
+        weakness L
+        pokePower "Water Cyclone", {
+          text "As often as you like during your turn (before your attack), you may move a [W] Energy card from your Active Pokémon to 1 of your Benched Pokémon. This power can't be used if Kingdra is affected by a Special Condition."
+          actionA {
+          }
+        }
+        move "Rapids", {
+          text "50 damage. Flip a coin. If heads, discard 1 Energy card attached to the Defending Pokémon, if any."
+          energyCost W, W, C, C
+          attackRequirement {}
+          onAttack {
+            damage 50
+          }
+        }
+      };
+      case LANTURN_20:
+      return evolution (this, from:"Chinchou", hp:HP080, type:W, retreatCost:1) {
+        weakness G
+        move "Lightning Ball", {
+          text "20 damage. "
```

Comment:
Extremely minor detail, but worth bringing up now while the iron is hot I guess :P

Since these may eventually be used in the UI according to one of the cards in trello, could a mass replace be used to remove that trailing space?

Replacing `0 damage. "` to `0 damage."` should cover these and nothing else iirc.

Reply:
With regular expressions you can be even more sure you're only changing what you want.

Reply:
I've removed the extra spaces

Reply:
My bad, missed to mention this happened in all new files 😅

Reply:
I removed it in Dragon, I didn't see it in any of the other new files